net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/Menu/Menu.xml

```diff
@@ -1,17 +1,19 @@
 <menu>
-  <Routing cssClass="fa fa-map-signs" order="45">
-    <General VisibleName="General" cssClass="fa fa-cog fa-fw" url="/ui/quagga/general/index" order="1"/>
-    <RIP VisibleName="RIP" cssClass="fa fa-expand fa-fw" url="/ui/quagga/rip/index" order="10" />
-    <OSPF VisibleName="OSPF" cssClass="fa fa-map fa-fw" url="/ui/quagga/ospf/index" order="20" />
-    <OSPFv3 VisibleName="OSPFv3" cssClass="fa fa-map fa-fw" url="/ui/quagga/ospf6/index" order="25" />
-    <!--<ISIS VisibleName="IS-IS" cssClass="fa fa-bolt fa-fw" url="/ui/quagga/isis/index" order="30" />-->
-    <BGP VisibleName="BGPv4" cssClass="fa fa-globe fa-fw" url="/ui/quagga/bgp/index" order="40" />
-    <Diagnostics VisibleName="Diagnostics" cssClass="fa fa-medkit fa-fw" order="50">
-      <General VisibleName="General" url="/ui/quagga/diagnostics/general" order="1" />
-      <OSPF VisibleName="OSPF" url="/ui/quagga/diagnostics/ospf" order="20" />
-      <OSPFv3 VisibleName="OSPFv3" url="/ui/quagga/diagnostics/ospfv3" order="25" />
-      <BGP VisibleName="BGPv4" url="/ui/quagga/diagnostics/bgp" order="40" />
-      <LOG VisibleName="Log" url="/ui/quagga/diagnostics/log" order="100" />
+  <!-- XXX for backwards compat sets order and cssClass! -->
+  <Routing order="35" cssClass="fa fa-wrench">
```

Comment:
wrench is probably not so ideal for routing. How about map-o? https://fontawesome.com/v4.7.0/icon/map-o

Reply:
wrench as in plubing.. there's no great icon. FRR is already using map. Maybe we should reshuffle?

Reply:
Ok, keep it as it is and I approve it, -o would have inverted colors so It would not interfere that much.